bin/assets/scripts/newform.js

```diff
@@ -22,3 +35,24 @@ code.addEventListener('keydown', (event) => {
     form.submit();
   }
 });
+
+window.addEventListener('drop', async (event) => {
+  const files = event.dataTransfer.files;
+  if (files.length !== 1) {
+    return;
+  }
```

Comment:
Une idée comme ça, mais pourquoi ne pas itérer sur chaque fichier et tenter de faire un bin différent à chaque fois ? (du coup ne pas rediriger mais renvoyer d'une manière ou d'une autre les différents liens générés)

Reply:
Je trouve que c'est pas à nous de dispatcher ses fichiers. Il n'a pas à mettre plusieurs fichiers, rtdbin != gist imo.